<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>donorEditor</class>
 <widget class="QWidget" name="donorEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelClientPhoto">
   <property name="geometry">
    <rect>
     <x>281</x>
     <y>12</y>
     <width>171</width>
     <height>191</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>128</width>
     <height>128</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MibitLineEdit" name="editClientPhone">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>238</y>
     <width>129</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>94</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="MibitLineEdit" name="editClientCell">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>294</y>
     <width>129</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>442</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>126</y>
     <width>58</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Address:</string>
   </property>
   <property name="buddy">
    <cstring>editClientAddress</cstring>
   </property>
  </widget>
  <widget class="QTextEdit" name="editClientAddress">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>149</y>
     <width>255</width>
     <height>60</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>60</height>
    </size>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>327</y>
     <width>39</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Since:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChangeClientPhoto">
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>225</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Change &amp;Photo</string>
   </property>
  </widget>
  <widget class="MibitLineEdit" name="editClientName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>93</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>14</y>
     <width>39</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Code:</string>
   </property>
   <property name="buddy">
    <cstring>editClientCode</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>43</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Name:</string>
   </property>
   <property name="buddy">
    <cstring>editClientName</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>215</y>
     <width>44</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone</string>
   </property>
   <property name="buddy">
    <cstring>editClientPhone</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>271</y>
     <width>75</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Cell Phone:</string>
   </property>
   <property name="buddy">
    <cstring>editClientCell</cstring>
   </property>
  </widget>
  <widget class="MibitLineEdit" name="editClientCode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>37</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="KDateWidget" name="sinceDatePicker">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>246</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDateWidget</class>
   <extends>QWidget</extends>
   <header>kdatewidget.h</header>
  </customwidget>
  <customwidget>
   <class>MibitLineEdit</class>
   <extends>QLineEdit</extends>
   <header>../../mibitWidgets/mibitlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
